src/figuresClasses.ts

```diff
@@ -1,11 +1,111 @@
-export interface Figure {}
+// export interface Figure {}
 
-export class Triangle implements Figure {}
+// export class Triangle implements Figure {}
 
-export class Circle implements Figure {}
+// export class Circle implements Figure {}
 
-export class Rectangle implements Figure {}
+// export class Rectangle implements Figure {}
 
-export function getInfo(figure): string {
-  return typeof figure;
+// export function getInfo(figure): string {
+//   return typeof figure;
+// }
+
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
Create a type for color and enum for shape 
```suggestion
enum Shape {
  Triangle = 'triangle',
  Circle = 'circle',
  Rectangle = 'rectangle',
}

type Color = 'red' | 'green' | 'blue';

export interface Figure {
  shape: Shape;
  color: Color;
```